app/build.gradle

```diff
@@ -161,8 +161,7 @@ dependencies {
 
     implementation 'net.ypresto.timbertreeutils:timbertreeutils:1.0.0'
 
-    // Timber
-    implementation 'com.jakewharton.timber:timber:4.7.1'
```

Comment:
We're still using `Timber` under the hood (just gaining some performance by calling it via `Slimber`), so lets keep this dependency around

Reply:
Timber is included in `com.github.PaulWoitaschek:Slimber:1.0.7`

Reply:
I know, but if we use a dependency directly we should be listing it as a direct dependency of our project. If not, we're relying on an implementation detail of our direct dependency, which might change when they do an update.